robot_modeling/DQ_SerialManipulatorDH.m

```diff
@@ -160,13 +116,15 @@
             
             % The optimized standard dh2dq calculation
             % Store half angles and displacements
-            half_theta = obj.theta(ith)/2.0;
-            d = obj.d(ith);
-            a = obj.a(ith);
-            half_alpha = obj.alpha(ith)/2.0;
+            half_theta = obj.dh_matrix_(1,ith)/2.0; %obj.theta(ith)/2.0; 
+            d = obj.dh_matrix_(2,ith); %obj.d(ith);
+            a = obj.dh_matrix_(3,ith); %obj.a(ith);
+            half_alpha = obj.dh_matrix_(4,ith)/2.0; %obj.alpha(ith)/2.0;
+            joint_type = obj.dh_matrix_(5,ith);
             
             % Add the effect of the joint value
-            if obj.type(ith) == obj.JOINT_ROTATIONAL
+            %if obj.type(ith) == obj.JOINT_ROTATIONAL
+            if joint_type == obj.JOINT_ROTATIONAL   
```

Comment:
I am a little confused since those lines of code were not added by me and were not proposed in the current PR. But I understand your comments. I'm going to implement the modifications aiming for better readability and avoiding recalculations.

Reply:
@bvadorno @juanjqo 

It's difficult for me to know what is the problem in question because all the files under discussion changed so much, but I believe we might be talking about the modifications for 20.04 discussed here:
https://github.com/dqrobotics/matlab/pull/56#issuecomment-619643404

More specifically, after review 
`3.1 Could you also break the lines at 80 characters to comply with our current guide style?`
the number of characters per line was adjusted but it would have been better to adjust the style to the current usage in MATLAB.